tests/e2e/core-tests/README.md

```diff
@@ -0,0 +1,100 @@
+# WooCommerce Core End to End Test Suite
+
+This package contains the automated end-to-end tests for WooCommerce.
+
+## Table of contents
+
+- [Pre-requisites](#pre-requisites)
+- [Setting up core tests](#setting-up-core-tests)
+- [Test functions](#test-functions)
+  - [Activation and setup](#activation-and-setup)
+  - [Merchant](#merchant)
+  - [Shopper](#shopper)
+- [Contributing a new test](#contributing-a-new-test)
+
+## Pre-requisites
+
+### Setting up the test environment
+
+Follow [E2E setup instructions](https://github.com/woocommerce/woocommerce/test/e2e/README.md). 
```

Comment:
This URL didn't work for me it should be `tests` instead of `test`, although even with that change it didn't resolve correctly. Could we use a relative link of `(../tests/e2e/README.md)` or does it need to be an absolute link for a package?

Reply:
@tammullen Thanks for catching that. I've pushed `72a121b` to fix. Can you merge this when you approve it?